mw-webapp/src/logic/usersTable/columns.tsx

```diff
@@ -0,0 +1,66 @@
+import {createColumnHelper} from "@tanstack/react-table";
+import {Link} from "src/component/link/Link";
+import {UserPreview} from "src/model/businessModelPreview/UserPreview";
+import {pages} from "src/router/pages";
+import {renderCellValue} from "src/utils/renderCellValue";
+
+const columnHelper = createColumnHelper<UserPreview>();
+
+/**
+ * Determines which columns will be in the table, the values in the cells and what types of data can be rendered in cells
+ * The tanstack table has a bug about typing columns:
+ * https://github.com/TanStack/table/issues/4382
+ * According to creators should only be using the column helper and not pre-typing columns
+ * We can add type as:
+ * ColumnDef<WayPreview, string & boolean & GoalPreview & userPreview & UserPreview[]>
+ * but it's not recommend by creators
```

Comment:
This example is not valid in this case

Reply:
How is this different from other columns.tsx file?

Reply:
You describe users table but there are example about wayPreview columns/ It's wrong